### PR TITLE
driver: add a new folder camera(to deprecate current usb_cam) for bazel build

### PR DESCRIPTION
**Pull request by jmtao**
…l we depracate the current one)

### CODE REVIEW

File: modules/drivers/usb_cam_new/nodes/usb_cam_wrapper.cc [hunk 1]
```diff
@@ -0,0 +1,273 @@
+/*********************************************************************
+*
+* Software License Agreement (TBD)
```
Comment (startcode) [inline modules/drivers/usb_cam_new/nodes/usb_cam_wrapper.cc]: Fix license

File: modules/drivers/usb_cam_new/nodes/usb_cam_nodelet.cc [hunk 2]
```diff
@@ -0,0 +1,52 @@
+/*********************************************************************
+*
+* Software License Agreement (TBD)
+*
+*  Copyright (c) 2017, XXX
```
Comment (startcode) [inline modules/drivers/usb_cam_new/nodes/usb_cam_nodelet.cc]: Fix license

File: modules/drivers/usb_cam_new/nodes/usb_cam_wrapper.h [hunk 3]
```diff
@@ -0,0 +1,103 @@
+/*********************************************************************
+*
+* Software License Agreement (TBD)
+*
+*  Copyright (c) 2017, XXX
```
Comment (startcode) [inline modules/drivers/usb_cam_new/nodes/usb_cam_wrapper.h]: Fix license

File: modules/drivers/usb_cam_new/nodes/usb_cam_wrapper.h [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+  std::string topic_name_;
+  std::string video_device_name_;
+  std::string io_method_name_;
+  std::string pixel_format_name_;
+  std::string camera_name_;
+  std::string camera_info_url_;
+
+  // std::string start_service_name_, start_service_name_;
+  // bool streaming_status_;
+  int image_width_;
```
Comment (startcode) [inline modules/drivers/usb_cam_new/nodes/usb_cam_wrapper.h]: initialize primary types inplace

File: modules/drivers/usb_cam_new/src/LICENSE [hunk 5]
```diff
@@ -0,0 +1,14 @@
+Video for Linux Two API Specification
+Revision 0.24
+Michael H Schimek  <mschimek@gmx.at>
+Bill Dirks
+Hans Verkuil
+Martin Rubli
```
Comment (startcode) [inline modules/drivers/usb_cam_new/src/LICENSE]: If this code is from another package, can we refer to the code by a library dependency?

File: modules/drivers/usb_cam_new/AUTHORS.md [hunk 6]
```diff
@@ -0,0 +1,9 @@
+Original Authors
```
Comment (startcode) [inline modules/drivers/usb_cam_new/AUTHORS.md]: 'usb_cam_new' is not a good folder name.

Comment (startcode) [reply modules/drivers/usb_cam_new/AUTHORS.md]: If this is a package written by others, import to Apollo as a dependency.

Comment (lianglia-apollo) [reply modules/drivers/usb_cam_new/AUTHORS.md]: Try to use another name, such as camera_driver ?

Comment (jmtao) [reply modules/drivers/usb_cam_new/AUTHORS.md]: It's a temporary folder name to co-exist with the current "usb_cam" folder, and will rename back to "usb_cam" once the current "usb_cam" is deprecated.

**Pull request merged**